opslevel/resource_opslevel_service_tag.go

```diff
@@ -155,12 +155,20 @@ func (serviceTagResource *ServiceTagResource) Read(ctx context.Context, req reso
 		serviceIdentifier = data.ServiceAlias.ValueString()
 		service, err = serviceTagResource.client.GetServiceWithAlias(serviceIdentifier)
 	}
-	if err != nil || service == nil {
+	if err != nil {
```

Comment:
Same as others where its a resource that depends on another resource being found first.